README.md

```diff
@@ -118,4 +118,4 @@ let myNavigation = new TenUp.navigation( '.navigation', {
 
 ## Demo
 
-An example implementation can be found in the `index.html` file in the root of this package.
+An example implementation can be found in the `demo/index.html`.
```

Comment:
The phrasing of this sentence seems off. Maybe something like this:

Example implementations can be found in the `demo` directory.